driver/terraform_download.go

```diff
@@ -189,7 +190,12 @@ func installTerraform(ctx context.Context, conf *config.TerraformConfig) (*goVer
 	// Create path if one doesn't already exist
 	os.MkdirAll(*conf.Path, os.ModePerm)
 
-	installedPath, err := tfinstall.ExactVersion(v.String(), *conf.Path).ExecPath(ctx)
+	installer := &releases.ExactVersion{
+		Product:    product.Terraform,
+		Version:    v,
+		InstallDir: *conf.Path,
+	}
+	installedPath, err := installer.Install(ctx)
```

Comment:
Do we have any tests that check that Terraform is installed on the expected path? Probably isn't necessary if we don't, but was just wondering!

Reply:
You are right! It looks like the benchmark and compatibility tests exercise the terraform download code and recently failed. I'll mention your comment on the fix for the benchmark and compatibility. It sounds like it would be good to have a test for this.